bin/install/install-optout-dcos-cli.sh

```diff
@@ -152,7 +152,7 @@ if [ -z "$DCOS_CLI_VERSION" ]; then
     if $(compare_version 1.6.1); then
         "$VIRTUAL_ENV_PATH/bin/pip" install --quiet "dcoscli<0.4.0"
     else
-        "$VIRTUAL_ENV_PATH/bin/pip" install --quiet "dcoscli"
+        "$VIRTUAL_ENV_PATH/bin/pip" install --quiet "dcoscli==0.4.12"
```

Comment:
The reason for this version is 1. versions afterwards use features that aren't compatible with dc/os 1.7 (which should be the last version to use non-binary clis). 2. a 2.7 bug got introduced after 0.4.12 :)